content/docs/languages/go/quickstart.md

```diff
@@ -129,7 +129,7 @@ Before you can use the new service method, you need to recompile the updated
 While still in the `examples/helloworld` directory, run the following command:
 
 ```sh
-$ protoc -I helloworld/ helloworld/helloworld.proto --go_out=plugins=grpc:helloworld
+$ protoc -I helloworld --go_out=helloworld --go_opt=paths=source_relative helloworld.proto
```

Comment:
This removed the `plugins=grpc` parameter -- isn't it needed to generate services?

Reply:
Oh thanks, I had ment to ask you about that. The command ran just fin w/o it, but now that I check the diff, the core of the generated file is missing (and on second thought it makes sense that `--go_out` doesn't necessarily imply that the `grpc` plugin is being used). Oops! I'll add it back.

Reply:
(And I was looking at https://developers.google.com/protocol-buffers/docs/reference/go-generated, that didn't include the plugin, but reading until the end of that page, I understand why.)

Reply:
Done. PTAL